parsons/salesforce/salesforce.py

```diff
@@ -0,0 +1,151 @@
+from simple_salesforce import Salesforce as _Salesforce
+from parsons.utilities import check_env
+# from parsons import Table
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class Salesorce:
+    """
+    `Args:`
+        username: str
+            The Salesforce username (usually an email address).
+            Not required if ``SALESFORCE_USERNAME`` env variable is passed.
+        password: str
+            The Salesforce password.
+            Not required if ``SALESFORCE_PASSWORD`` env variable is passed.
+        security_token: str
+            The Salesforce security token that can be acquired or reset in
+            Settings > My Personal Information > Reset My Security Token.
+            Not required if ``SALESFORCE_SECURITY_TOKEN`` env variable is passed.
+        domain: str
+            If the Saleforce instance is a sandbox, set to `'test'`. Otherwise, not required.
+            Can also be passed through ``SALESFORCE_DOMAIN`` env variable.
```

Comment:
Might it make more sense to have the argument be `test_environment` boolean or something like that if that is the only instance in which it might be set?

Reply:
Please add the `test_environment` arg to the doc strings.